redis/src/cluster.rs

```diff
@@ -94,6 +80,20 @@ impl TlsMode {
     }
 }
 
+/// This is a connection of Redis cluster.
```

Comment:
This is an improvement, but I'd prefer to instead move `TlsMode` and its `impl` closer to the bottom of the module.

Reply:
Yes, moving `SlotMap` to the bottom makes sense to me as well.

Reply:
👍